tests/all_tests_v2.nim

```diff
@@ -11,7 +11,8 @@ import
   ./v2/test_utils_pagination,
   ./v2/test_message_store_queue,
   ./v2/test_message_store_queue_pagination,
-  ./v2/test_message_store,
+  ./v2/test_message_store_sqlite_query,
+  ./v2/test_message_store_sqlite,
```

Comment:
May be good (in future) to have general `MessageStore` tests that can instantiate both the `WakuMessageStore` and `StoreQueue` implementations (and whatever else we add in future) and compare query results.

Reply:
That sounds like a good idea to me 😁 
 
Indeed, I would like to completely decouple the WakuStore protocol from the two kinds of stores (in-memory aka queue, and the SQLite store-backed store) and just provide one ref to the protocol implementation. 

Then if we want to do some caching in-memory and back it with persistent storage for speed, what I would do is implement a wrapper method that handles that logic.


At this point you might be thinking:

> And why did you not do it yet?

I would have done it but... the "resume store" functions in the WakuStore protocol need some cleanup before starting to do some refactoring in that direction. It is not straightforward to replace it and also the interface might need some extra method to support the resume functionality.

So... one step after the other 😁

Reply:
Hahaha. I wasn't thinking that (given the size of the open PRs), but thanks for clarifying. :D